packages/setup-remote-server/lib/commands/server.js

```diff
@@ -0,0 +1,475 @@
+				.replace( '~/', `/home/${ hostUser }/` )
+				.replace( '{{application}}', deployYMLData[ '.base' ].application )
+				.replace( '{{stage}}', envoirnment ),
+			message: 'Enter the path for the site directory:',
+			validate: validatePath,
+		},
+	] );
+
+	log( format.warning( 'Create a new database on the hosting provider.' ) );
+
+	await recursiveConfirm( {
+		type: 'confirm',
+		name: 'createdDB',
+		message: 'Have you created the database?',
+		default: false,
+	} );
+
+	const { dbHost, dbNmae, dbUser, dbPassword } = await inquirer.prompt( [
+		{
+			type: 'input',
+			name: 'dbHost',
+			default: 'localhost',
+			message: 'Enter the database host:',
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'dbNmae',
+			message: 'Enter the database name:',
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'dbUser',
+			message: 'Enter the db username:',
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'dbPassword',
+			message: 'Enter the database password:',
+			validate: validateNotEmpty,
+		},
+	] );
+
+	const tempEnvFile = `.local-server/.env.${ envoirnment }`;
+	try {
+		fs.copyFileSync( '.local-server/.env', tempEnvFile );
+		log( format.success( `local-server/.env copied to ${ tempEnvFile }` ) );
+	} catch ( error ) {
+		log( error );
+	}
+
+	const dotenvConfig = dotenv.parse( fs.readFileSync( `${ dotLocalServer }/.env` ) );
+
+	let httpHost = dotenvConfig.URL_STAGING;
+	if ( envoirnment === 'production' ) {
+		httpHost = dotenvConfig.URL_PRODUCTION;
+	}
+
+	// Rename files in local checkout.
+	await runStep( 'Renaming project files', 'Could not rename files.', async () => {
+		const envReplacementOptions = {
+			files: [ `${ WORKING_DIR }/${ tempEnvFile }` ],
+			from: [
+				/WP_ENV=development/g,
+				new RegExp( `_HTTP_HOST="${ dotenvConfig._HTTP_HOST }"`, 'g' ),
+				/DB_HOST=\${MYSQL_HOST}/g,
+				/DB_NAME=\${MYSQL_DATABASE}/g,
+				/DB_USER=\${MYSQL_USER}/g,
+				/DB_PASSWORD=\${MYSQL_PASSWORD}/g,
+				/WP_DEBUG_DISPLAY=true/g,
+				/SCRIPT_DEBUG=true/g,
+			],
+			to: [
+				`WP_ENV=${ envoirnment }`,
+				`_HTTP_HOST="${ httpHost.replace( 'https://', '' ) }"`,
+				`DB_HOST=${ dbHost }`,
+				`DB_NAME=${ dbNmae }`,
+				`DB_USER=${ dbUser }`,
+				`DB_PASSWORD=${ dbPassword }`,
+				'WP_DEBUG_DISPLAY=false',
+				'SCRIPT_DEBUG=false',
+			],
+		};
+		await replace( envReplacementOptions );
+
+		if ( envoirnment === 'staging' ) {
+			const stagingEnvReplacementOptions = {
+				files: [ `${ WORKING_DIR }/${ tempEnvFile }` ],
+				from: [ /JETPACK_DEV_DEBUG=true/g ],
+				to: [ 'JETPACK_STAGING_MODE=true' ],
+			};
+			await replace( stagingEnvReplacementOptions );
+		}
+
+		if ( envoirnment === 'production' ) {
+			const prodEnvReplacementOptions = {
+				files: [ `${ WORKING_DIR }/${ tempEnvFile }` ],
+				from: [
+					/WP_DEBUG=true/g,
+					/WP_DEBUG_LOG=true/g,
+					/SAVEQUERIES=true/g,
+					/QM_DISABLED=false/g,
+					/JETPACK_DEV_DEBUG=true/g,
```

Comment:
Not directly related to this but I think it's time to remove any Jetpack related configs since we haven't used Jetpack in a new project for a while now.

Reply:
I was thinking the same thing. Will have to remove from the boilerplate too.

Reply:
Do it!